--- origsrc/texinfo-6.4.91/info/pseudotty.c	2016-01-31 06:42:56.000000000 -0500
+++ src/texinfo-6.4.91/info/pseudotty.c	2017-09-01 11:05:21.981139000 -0400
@@ -64,11 +64,12 @@ main (int argc, char *argv[])
     exit (1);
   error (0, 0, "%s", name);
 
-#ifdef HAVE_STROPTS_H
   error (0, 0, "opening slave device");
   slave = open (name, O_RDWR);
   if (slave == -1)
     exit (1);
+
+#ifdef HAVE_STROPTS_H
   if (!isatty (slave))
     {
       error (0, 0, "performing STREAMS ioctl's on slave");
